packages/ui/src/components/sign-in/sign-in.tsx

```diff
@@ -226,6 +226,7 @@ export function SignInComponentLoaded() {
                             <LinkButton
                               size='sm'
                               disabled={isGlobalLoading}
+                              type='button'
```

Comment:
Fixes issue where hitting enter button would take you to forgot password flow vs submitting the password form.